### PR TITLE
Added the Microsoft E company store

### PR DESCRIPTION
**Pull request by Darkgran1**
Added the Microsoft E company store website and screenshot

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1130,6 +1130,16 @@ the Chrome extension Don't Fuck With Paste.
 
 |MetLife|
 
+`Microsoft (e company store) <https://store.ecompanystore.com/microsoftevents/Shop/Register/PASSHOLDER#/>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Max of 16 character oh and please don't use any characters we don'y know how to escape properly 
+also if it starts with ? you may break our shitty website. What out with your password generator
```
Comment (duffn) [inline README.rst:1137]: Can you try to keep this clean please?

Comment (Darkgran1) [reply README.rst:1137]: Happy to make the change just didn't think it mattered given the example directly above

Comment (duffn) [reply README.rst]: Thanks, I may indeed have missed some.

**Comment by duffn** [comment]
Thank you for the contribution!

**Pull request merged**